docs/conf.py

```diff
@@ -245,3 +247,22 @@ def linkcode_resolve(domain, info):
         }
     }
 }
+
+
+def check_if_v07():
+    """Check to see if current version being built is > v0.7."""
+    result = False
+
+    if parse(fairlearn.__version__) > parse("0.7"):
+        print("Detected version > 0.7 in fairlearn.__version__")
+        result = True
+
+    return result
+
+
+# Setup for sphinx-bibtex
+
+# Only use sphinx-bibtex if version is above 0.7
```

Comment:
It's not used in other versions, so it should just be fine(?) or am I missing something?

Oh or is this about the refs file not existing?

Just trying to make sure I get it 🙂

Reply:
Yes, you are right, it will be fine. However, sphinx-multiversion uses the latest `conf.py` and for other versions we don't have a `refs.bib` file which generates a warning. Although it will not stop the documentation build, this can worry some new users, so I made it explicit to use `sphinx-bibtex` only when the required version is available.

![image](https://user-images.githubusercontent.com/50991099/156820471-33e3abee-63b1-41b4-bbdb-111dea3c41b6.png)
It is really minor but it looks something like this.